### ISSUE TITLE
[Clarification Needed] BE BUG: https://rhapp-backend-devel.rhdevs.repl.co/social/user allows users to be created with duplicate userIDs

### ISSUE THREAD

**Issue by eugenetaan** [rhdevs/rhapp#569]
duplicate users can be created with the same userID different passwordHash (POST request)

**Comment by eugenetaan** [comment]
for POST request

**Comment by woowenjun99** [comment]
<img width="577" alt="Screenshot 2022-01-30 at 1 32 24 PM" src="https://user-images.githubusercontent.com/88195289/151688057-927b47b5-9068-4d88-9623-10547a814440.png">
<img width="469" alt="Screenshot 2022-01-30 at 1 34 01 PM" src="https://user-images.githubusercontent.com/88195289/151688062-8bb7ba04-9d8f-469c-a2ca-bc490bf9b095.png">
<img width="803" alt="Screenshot 2022-01-30 at 1 45 17 PM" src="https://user-images.githubusercontent.com/88195289/151688254-88eb426d-a0b0-4bb8-b0a5-453c50bfa3d2.png">

Correct me if I am wrong, I tried to test this route and method but there seemed to be a validation error. I checked with MongoDB compass and saw that it matches the sample code of validation that failed. Is it supposed to be like this?
